EDU-2191-Component-EN-Edge-Firewall-Rules-Engine (#343)
aziontech/docs: src/content/docs/en/pages/products/edge-firewall/rules-engine.mdx

An Edge Firewall Rule Set can have as many rules as needed. These rules are also interchangeable, as you can share the same rule for all other Edge Firewall configurations.

> :::note
> Learn more about [Edge Firewall](/en/documentation/products/edge-firewall/) and its modules.
> :::

Reviewer: ```suggestion
import Button from '~/components/Button.astro';

<Button href="/en/documentation/products/edge-firewall/" text="go to Edge Firewall reference" variant="secondary"> 

</Button>
```

Reviewer: In these cases, we're using the button component. You need to import it once, and you can add one or more buttons.

*Criteria* determines the set of conditions that need to be met for the execution of *Behaviors*. The availability of criteria depends on the enabled modules of your Edge Firewall. 

> :::note
> Learn more about [Edge Firewall modules](/en/documentation/products/edge-firewall/#how-azion-edge-firewall-works).
> :::

Reviewer: ```suggestion
<Button href="/en/documentation/products/edge-firewall/#how-azion-edge-firewall-works" text="go to Edge Firewall modules reference" variant="secondary"> 
</Button>
```

Reviewer: idem

| exists              | The variable has a defined value. For example, Request Args exists if an argument is sent in the request's query string. | - |
| does not exist      | The variable doesn't have a defined value. For example, Request Args doesn't exist if an argument is sent in the request's query string. | - |

> :::tip

Reviewer: ```suggestion
:::note
```

These operators may vary according to the selected Criteria.
:::

#### Logic Operators

Multiple conditions can be defined using the logical operators `and` and `or`. 

> :::tip

Reviewer: ```suggestion
:::note
```

| Run a Function                | It runs a function specified as a parameter. The function must have been previously instantiated and parameterized in the Functions tab in order to be used. | Edge Functions           |
| Set Custom Response           | It allows a customized response when the request matches the criteria. You can customize the `Status Code` by changing it from 200 to 499, `Content Type` header, and `Content Body` of your request with a maximum of `500` characters. | - | 

> :::tip

Reviewer: ```suggestion
:::note
```
